storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -94,11 +170,27 @@ Een manier om dat te doen is met de volgende _css snippet_, waarin `hyphens` en
 
 Voor verdere informatie: [Wrapping and breaking text op MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text)
 
-## Hoe te gebruiken?
+## Het lettertype installeren
+
+Als je de React-componenten gebruikt hoef je niets te doen.
+Deze gebruiken allemaal het juiste lettertype.
+
+In andere gevallen installeer je Amsterdam Sans op 1 van deze 2 manieren:
+
+1. Kopieer [deze regels CSS](https://amsterdam.nl/publish/pages/1007152/amsterdam_sans_v_0_9.css) naar je applicatie.
+   Het maakt gebruik van ons snelle CDN.
```

Comment:
Fast CDN? They are uploads to amsterdam.nl, which will be changing pretty soon. If we use an actual CDN with proper headers for the font files we can change the imports in the package and link this to the [npm code view](https://www.npmjs.com/package/@amsterdam/design-system-assets?activeTab=code). Or just use a code block in this document.

Reply:
No idea, I copied this from the old site. Should have opened the link :)

What do you say, remove it and just suggest going with our package?

Reply:
Not sure if downloading our font is following the license policy, from a CDN we would have some form of control, but I guess you can never really stop anyone from downloading/converting the font.

Reply:
I think so too. I’ve updated the text.